src/ccs/cccs.rs

```diff
@@ -28,47 +28,64 @@ use std::sync::Arc;
 
 use super::util::compute_sum_Mz;
 use super::util::virtual_poly::VirtualPolynomial;
-use super::{CCSShape, CCSWitness};
+use super::CCS;
 
 /// A type that holds the shape of a Committed CCS (CCCS) instance
-#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(bound = "")]
-pub struct CCCSShape<G: Group> {
-  // Sequence of sparse MLE polynomials in s+s' variables M_MLE1, ..., M_MLEt
-  pub(crate) M_MLE: Vec<MultilinearPolynomial<G::Scalar>>,
-
-  pub(crate) ccs: CCSShape<G>,
+#[derive(Clone, Debug, PartialEq, Eq /*Serialize, Deserialize*/)]
+//#[serde(bound(deserialize = "'de: 'a"))]
+pub struct CCCS<G: Group> {
+  // The `z` vector used as input for this instance.
+  pub(crate) z: Vec<G::Scalar>,
+  // Commitment to the witness of `z`.
+  pub(crate) w_comm: Commitment<G>,
```

Comment:
How come we have two commitments here now when there was just one in CCCSInstance before? Perhaps more clear reading code further down

Reply:
Considering the `z` vector which contains the public inputs & witnesses of the current CCCS, we have 3 components: `(1,x,w)`.
We store a commit to `x` and `w`. And the commitment to `w` is needed for the folding.

But this made me realize that the commitment to `x` is not needed. Hence I removed it.

Reply:
Fixed in 52f6a62